client/src/AllVideos.css

```diff
@@ -0,0 +1,199 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+.all-videos-wrapper {
+    width: 100%;
+    margin: 80px 0;
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+@media only screen and (min-width: 375px) {
+    .all-videos-wrapper {
+        width: 100%;
+        margin: 120px 0;
+        list-style: none;
+        display: flex;
+        flex-direction: column;
+        gap: 20px;
+    } 
+}
+
+.all-videos-title {
+    color: white;
+    text-align: center;
+    margin: 0 20px 0 20px;
+    padding-bottom: 20px;
+    border-bottom: .5px solid #fcd277;
+}
+
+@media only screen and (min-width: 375px) {
+    .all-videos-title {
+        margin: 0 65px 50px 65px;
+        margin-bottom: 50px;
+    }
+}
+
+.all-videos-filter-wrapper {
+    display: flex;
+    gap: 5px;
+    padding-left: 100px;
+} 
+
+.all-videos-filter-btn {
+    background-color: transparent;
+    color: white;
+    border: none;
+}
+
+@media only screen and (min-width: 375px) {
```

Comment:
It doesn't look like these properties are being changed, can remove the media query?

Reply:
Sure thing.